tst/EnergyPlus/unit/SwimmingPool.unit.cc

```diff
@@ -255,3 +257,127 @@ TEST_F(EnergyPlusFixture, SwimmingPool_InitSwimmingPoolPlantNodeFlow)
     EXPECT_EQ(DataSizing::CompDesWaterFlow(1).DesVolFlowRate, 0.002);
 
 }
+
+TEST_F(EnergyPlusFixture, SwimmingPool_ErrorCheckSetupPoolSurfaceTest)
+{
+    
+    // Tests for InitSwimmingPoolPlantLoopIndex
+    SwimmingPool::clear_state();
+    DataSurfaces::clear_state();
+    DataHeatBalance::clear_state();
```

Comment:
Clear_states do not belong in unit tests. They are handled by the test fixture.

Reply:
@rraustad Got it.  Removed and I'll push that commit up once energyplus_test runs successfully.

Reply:
Done.  These lines have been removed.  Energyplus_test still runs completely as expected with no errors.  A new commit has now been made for these changes.